api/src/main/java/org/openmrs/module/appointmentscheduling/AppointmentType.java

```diff
@@ -33,9 +36,10 @@ public AppointmentType(Integer appointmentTypeId) {
 		setId(appointmentTypeId);
 	}
 	
-	public AppointmentType(String name, String description, Integer duration) {
+	public AppointmentType(String name, String description, Integer duration, VisitType visitType) {

```

Comment:
Instead of modifying the existing constructor, we should just add a new constructor that adds visit type for backwards compatibility.